CHANGELOG.next.toml

```diff
@@ -36,6 +36,18 @@ references = ["aws-sdk-rust#1079"]
 meta = { "breaking" = false, "bug" = true, "tada" = false }
 author = "rcoh"
 
+[[aws-sdk-rust]]
+message = "Fixes stalled upload stream protection to not apply to empty request bodies and to stop checking for violations once the request body has been read."
+references = ["aws-sdk-rust#1141", "aws-sdk-rust#1146", "aws-sdk-rust#1148"]
```

Comment:
Will `aws-sdk-rust#1141` (for CopyObject) and `aws-sdk-rust#1146` (for MultipartUpload) be addressed by this PR?

Reply:
CopyObject is going to be resolved in a separate PR that I linked in an earlier post. MPU should be fixed by this PR.